src/main/scala/akka/wamp/package.scala

```diff
@@ -55,7 +55,7 @@ package object wamp {
 
   final object Uri {
     // strict URI check disallowing empty URI components
-    val regex = "^([0-9a-z_]+\\.)*([0-9a-z_]+)$".r
+    val regex = "^([0-9a-zA-Z_]+\\.)*([0-9a-zA-Z_]+)$".r
```

Comment:
There are WAMP servers which need the topic contain upper case. As this might not be very well specified in the spec it's a real world issue that I need to overcome.

I think a lot of people will just their code with AutobahnJs client and if it works they would assume it's good for everyone.